apps/trading-e2e/src/integration/capsule.cy.ts

```diff
@@ -113,35 +120,41 @@ describe('capsule', { tags: '@slow' }, () => {
           checkIfDataAndTimeOfCreationAndUpdateIsEqual(orderCreatedAt);
         });
     });
-    //edit order
+  });
+
+  it('can edit order', function () {
     cy.getByTestId(ordersTab).click();
     cy.getByTestId('edit').first().should('be.visible').click();
     cy.getByTestId('dialog-title').should('contain.text', 'Edit order');
-    cy.get('#limitPrice').focus().clear().type('200');
+    cy.get('#limitPrice').focus().clear().type(newPrice);
     cy.getByTestId('edit-order').find('[type="submit"]').click();
     cy.getByTestId(toastContent).should(
       'contain.text',
-      `ConfirmedYour transaction has been confirmed View in block explorerEdit order - activeTEST.24h+0.0005 @ 200.00 ${usdcSymbol}+0.0005 @ 200.00 ${usdcSymbol}`
+      `ConfirmedYour transaction has been confirmed View in block explorerEdit order - activeTEST.24h+${size} @ ${price}.00 ${usdcSymbol}+${size} @ ${newPrice}.00 ${usdcSymbol}`,
+      { matchCase: false }
     );
     cy.getByTestId(ordersTab).click();
-    cy.getByTestId(toastCloseBtn).click();
+    cy.getByTestId(toastCloseBtn).click({ multiple: true });
     cy.get('.ag-center-cols-container')
       .children()
       .first()
       .within(() => {
         cy.get(`[col-id='${orderPrice}']`).then(($price) => {
-          expect(parseFloat($price.text())).to.equal(parseFloat('200'));
+          expect(parseFloat($price.text())).to.equal(parseFloat(newPrice));
         });
         checkIfDataAndTimeOfCreationAndUpdateIsEqual(orderUpdatedAt);
       });
-    //cancel order
+  });
+
+  it('can cancel order', function () {
     cy.getByTestId(ordersTab).click();
     cy.getByTestId('cancel').first().click();
     cy.getByTestId(toastContent).should(
       'contain.text',
-      `ConfirmedYour transaction has been confirmed View in block explorerCancel order - cancelledTEST.24h+0.0005 @ 200.00 ${usdcSymbol}`
+      `ConfirmedYour transaction has been confirmed View in block explorerCancel order - cancelledTEST.24h+${size} @ ${newPrice}.00 ${usdcSymbol}`,
```

Comment:
The idea is to check if tests will be stable with checking the whole text and then we can be sure that we check everything. If the tests are not stable because the text will change, we will do it separately.